components/browser/menu/src/main/res/layout/mozac_browser_menu_item_checkbox.xml

```diff
@@ -2,13 +2,33 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.appcompat.widget.AppCompatCheckBox xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/Mozac.Browser.Menu.Item.Text"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@android:style/TextAppearance.Material.Menu"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="?selectableItemBackground"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/mozac_browser_menu_item_container_layout_height"
-    android:button="@null"
-    android:drawableEnd="?android:attr/listChoiceIndicatorMultiple"
-    android:drawablePadding="12dp"
+    android:layout_height="48dp"
```

Comment:
Does it make sense to use a minimum height instead and expand if the text overflows? I'm not sure what the native menu does here.

Alternatively, maybe `@dimen/mozac_browser_menu_item_container_layout_height` should be used?